SparkPost/2web2ui
# SA-2053: Fix signals health score for mailbox provider

### What Changed

- Fixed Mailbox Provider facet for Health Score due to new formatted names.
- Removed formatter for mb health score calls in API
- Encoded the url for mb provider to account for strange formatted names.

### How To Test

- Go to `/signals/health-score/`
- In the table, set facet filter to mailbox provider.
- Click on a mailbox provider (Do multiple with strange names such as Hotmail / Outlook & Verizon Media Group(.
- Verify that the heath score page works.
